plugins/layout/layout.cpp

```diff
@@ -204,6 +205,33 @@ class BoxSizerComponent : public ComponentBase
 	}
 };
 
+class WrapSizerComponent : public ComponentBase
+{
+public:
+	wxObject* Create(IObject *obj, wxObject * /*parent*/)
+	{
+		wxWrapSizer *sizer = new wxWrapSizer(obj->GetPropertyAsInteger(_("orient")));
```

Comment:
Try to align the `*` with the type like you did with the return type of `Create`.